paparazzi/src/main/java/app/cash/paparazzi/SnapshotHandler.kt

```diff
@@ -28,3 +30,23 @@ public interface SnapshotHandler : Closeable {
 
 public fun detectMaxPercentDifferenceDefault(): Double =
   System.getProperty("app.cash.paparazzi.maxPercentDifferenceDefault")?.toDoubleOrNull() ?: 0.01
+
+/**
+ * Returns a LocaleQualifier for snapshots.
+ *
+ * Uses the provided `locale` if passed in, otherwise falls back to the
+ * `app.cash.paparazzi.localeDefault` system property, and finally the
+ * default `LocaleQualifier()` if neither are available.
+ *
+ * @param locale Optional Locale to use for snapshot tests.
+ * @return LocaleQualifier for snapshot tests.
+ */
+internal fun detectLocaleQualifierDefault(locale: String?): LocaleQualifier {
```

Comment:
why is this method in SnapshotHandler?  seems like a companion object method in DeviceConfig would be a better home?

Reply:
arguably, `detectMaxPercentDifferenceDefault` shouldn't be there either 😅